### PR TITLE
[更新完毕] tensor_matrix

### PR DESCRIPTION
**Pull request by isdanni**
Related issue: #82

### CODE REVIEW

File: locales/zh_CN/LC_MESSAGES/tensor_matrix.po [hunk 1]
```diff
@@ -46,32 +50,32 @@ msgid "Every global variable is an N-dimensional tensor."
...
 
 #: ../../tensor_matrix.rst:25
 msgid "Tensor values are initially zero."
-msgstr "张量的元素值全部会被初始化为0。"
+msgstr "张量元素全部会被初始化为0。"
 
 #: ../../tensor_matrix.rst:26
 msgid "Sparse tensors are initially inactive."
-msgstr "稀疏张量的元素全部会被初始化为 *未激活* 。"
+msgstr "稀疏张量最初是未激活的。"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/tensor_matrix.po]:
```suggestion
msgstr "稀疏张量的元素最初是全部未激活的。"
```

File: locales/zh_CN/LC_MESSAGES/tensor_matrix.po [hunk 2]
```diff
@@ -82,132 +86,119 @@ msgid "Tensors of matrices"
...
-"suggest using only small matrices. For example, ``2x1``, ``3x3``, ``4x4``"
-" matrices are fine, yet ``32x6`` is probably too big as a matrix size."
+"For performance reasons matrix operations will be unrolled, therefore we suggest "
+"using only small matrices. For example, ``2x1``, ``3x3``, ``4x4`` matrices are "
+"fine, yet ``32x6`` is probably too big as a matrix size."
 msgstr ""
-"由于性能原因，矩阵操作会在 *编译时展开* ，因此我们只建议用小的矩阵。比如 ``2x1``, ``3x3``, ``4x4`` 的矩阵是合理的，"
-" 然而 ``32x6`` 恐怕对于矩阵来说太大了。"
+"由于性能原因，矩阵运算将被展开，因此我们建议仅使用小型矩阵。 例如，``2x1``， "
+"``3x3`` ， ``4x4`` 矩阵很好，但 ``32x6`` 可能太大了。"
```
Comment (archibate) [inline locales/zh_CN/LC_MESSAGES/tensor_matrix.po]:
```suggestion
"``3x3`` ， ``4x4`` 矩阵还好，但 ``32x6`` 可能太大了。"
```

**Review by archibate — COMMENTED** [review]
Thank for the quick response! I only have nits. @yuanming-hu Could you give @isdanni write-access to this repo? I'm more occupied in taichi main repo and homework and may hardly have time reviewing & merging other's contribution, it would be great if someone could take part of my responsibility.

**Comment by yuanming-hu** [comment]
> Thank for the quick response! I only have nits. @yuanming-hu Could you give @isdanni write-access to this repo? I'm more occupied in taichi main repo and homework and may hardly have time reviewing & merging other's contribution, it would be great if someone could take part of my responsibility.

Sure, now @isdanni  and @ArkhamWJZ should have write access to this repo. Thanks for all the work happening here! Chinese users will greatly benefit from the translated documentation in their native language :-)

(Sorry about my delayed reply.)

**Pull request merged**